recipes/mmdet/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "mmdet" %}
+{% set version = "2.25.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/open-mmlab/mmdetection/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6642d593395a6c2d13bd1a574525477a8d08ef0fb7e67f9127a91b67193b06a9
+
+build:
+  #noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
```

```suggestion
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv
```

this is fine. Just let the reviewer know why it fails on windows. otherwise no issue. users of conda-forge will not be able to install this package on windows.

---

changed it this way a fraction of second before you submitted it :) thanks!